core/core.go

```diff
@@ -296,7 +297,22 @@ func getBazelVersion() (string, error) {
 		}
 	}
 
-	return "latest", nil
+	fallbackVersion := os.Getenv("USE_BAZEL_FALLBACK_VERSION")
```

Comment:
is there any downside to accepting USE_BAZEL_FALLBACK_VERSION from the .bazeliskrc file, too?

Reply:
That sounds fine. I didn't deal with `.bazeliskrc` interpretation here because the current code doesn't, and I think that should be implemented in a separate PR.

Reply:
I'll note that this means we read the `.bazeliskrc` file up to twice in this function, which is pretty alright for now but it should probably only be read once if `USE_BAZEL_NIGHTLY` or `USE_BAZEL_CANARY` get implemented.